src/pages/SignUp/SignUp3.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { Component } from 'react';
+import Footer from '../../components/Footer/Footer';
+import './SignUp3.scss';
+
+class SignUp3 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      mobile: '',
+      year: '',
+      month: '',
+      date: '',
+      pw: '',
+      pwCheack: '',
+      gender: '',
+    };
+  }
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  SignUp = () => {
+    fetch('http://10.58.2.219:8000/postings/posting', {
+      method: 'POST',
+      body: JSON.stringify({
+        name: this.state.name,
+        email: this.state.email,
+        mobile: this.state.mobile.replace(/\-/g, ''),
+        birthDay:
+          this.state.year + '-' + this.state.month + '-' + this.state.date,
+        Password: this.state.pw,
+        passwordCheck: this.state.pwCheck,
+      }),
+    })
+      .then(response => response.json())
+      .then(response => {
+        if (response.message === 'SUCCESS') {
+          alert('성공');
+        } else if (response.message === 'INVALID_USER') {
+          alert('이메일과 비밀번호를 다시 확인해주세요');
+        }
+      });
+  };
+
+  render() {
+    console.log(this.state);
+    const { name, email, mobile, year, month, date, pwCheack, gender } =
+      this.state;
+
+    const cheackLoginCall = () => {
+      cheackLogin(this.state.pw);
+    };
+
+    const cheackLogin = pw => {
```

Comment:
```suggestion
    const checkLogin = () => {
```
- 인자로 받지 않고 곧장 사용할 수도 있지 않을까요?
- check에 오타있습니다

Reply:
넵 !!!